members/anand/index.md

```diff
@@ -2,4 +2,7 @@
 
 <!--# Publications -->
 
-<!-- \publist{https://arxiv.org/a/soejima_t_1.html} -->
\ No newline at end of file
+<!-- \publist{https://arxiv.org/a/soejima_t_1.html} -->
+
+\publist{https://arxiv.org/a/anand_s_1.html}
```

Comment:
sorry my README wasn't clear. you need either your profile page or your search result, but not both!

Reply:
Taige has both in his `index.md`, so I guess it doesn't break things.

Reply:
yes, it just duplicates your publication list.

Reply:
and Taige's `index.md` has already been corrected.